worldengine/cli/main.py

```diff
@@ -395,9 +394,12 @@ def main():
 
     if args.seed is not None:
         seed = int(args.seed)
+        if not 0 <= seed <= 65535:  # platec might not be able to handle integers larger than uint16; TODO: check this
+            print("Seed has to be in the range of 0 and 65535.")
```

Comment:
Thanks a lot. I will add some kind of comment since the restricted range does seem a bit odd on first glance.